app/controllers/TagManagementApi.scala

```diff
@@ -140,4 +141,11 @@ object TagManagementApi extends Controller with PanDomainAuthActions {
     Ok(Json.toJson(jobs))
   }
 
+  def getTagsAsXml() = APIAuthAction.async {
+    Future(TagRepository.loadAllTags.seq.map(_.asXml)) map { tags =>
```

Comment:
We have a cached version of this which would be alot faster. I don't think there's currently a method to access it though.